src/main/main.js

```diff
@@ -99,30 +93,22 @@ function onReady(app) {
 			appUpdater();
 		}
 		await db.init();
-		if (!isDevMode() && !isTestMode()) {
-			await crashReportService.startCrashReport();
-		}
 		const store = configureStore(global.state, 'main');
+		const container = configureContrainer(store, app);
+		const di = container.cradle;
```

Comment:
@sk91 would make more sense to call this `applicationContext` instead of di? Thoughts?

Reply:
Also, there is a typo on method name and I think it should be `configureContainer`. Or if we are calling applicationContext we could call it `configureContext`. Thoughts?